commodore/component/compile.py

```diff
@@ -90,6 +92,8 @@ def compile_component(config: Config, component_path, value_files, search_paths,
 
         # Fetch Jsonnet libs
         fetch_jsonnet_libs(config, libs)
+        if os.path.exists(os.path.join(component_path, 'jsonnetfile.json')):
```

Comment:
Since `component_path` is already a path type, it should be possible to do
```suggestion
        if (component_path / 'jsonnetfile.json').exists():
```

Reply:
looks like the suggestion was not fully applied

Reply:
```suggestion
        if (component_path / 'jsonnetfile.json').exists():
```